tf_patches/cudnn_int8x32.diff

```diff
@@ -0,0 +1,563 @@
+diff --git a/tensorflow/compiler/xla/debug_options_flags.cc b/tensorflow/compiler/xla/debug_options_flags.cc
```

Comment:
Can we add a TO_BE_REMOVED comment? I am concerned about the increasing number of diffs we carry.. These kind of cuda error usually will be fixed by someone after a couple days, it is fine for us to carry the patch for a short period of time.

Reply:
Yeah, I'll add in a comment before I submit. Holding off on pushing another commit until the build passes or fails